src/pages/taskQueue/TaskQueueTable.tsx

```diff
@@ -87,7 +87,7 @@ export const TaskQueueTable = () => {
                 taskQueueAnalytics.sendEvent({ name: "Click Task Link" })
               }
             >
-              {displayName}
+              <WordBreak>{displayName}</WordBreak>
```

Comment:
Can you make the same change to version ids I commented on the ticket.

Reply:
sure thing